src/ngraph/op/fused/elu.cpp

```diff
@@ -52,9 +54,9 @@ NodeVector op::Elu::decompose_op() const
 
 shared_ptr<Node> op::Elu::copy_with_new_args(const NodeVector& new_args) const
 {
-    if (new_args.size() != 2)
+    if (new_args.size() != 1)
```

Comment:
Since you're already modifying this line, can you use `check_new_args_count` for consistency with other ops?